platform/broadcom/docker-syncd-brcm/start.sh

```diff
@@ -8,11 +8,22 @@ SYNCD_SOCKET_FILE=/var/run/sswsyncd/sswsyncd.socket
 rm -f ${SYNCD_SOCKET_FILE}
 
 mkdir -p /etc/sai.d/
+rm /etc/sai.d/*
```

Comment:
not sure you should delete every file here.

Reply:
The reason I am clearing it up is because in my manual test there were leftover generated config.bcm file and it is easier for me to debug in case things did not work and I can tell where the issue occurred if I cleaned it up every time.  
I am removing this since you are not comfortable with it.